README.md

```diff
@@ -221,6 +221,15 @@ You can control whether new users can sign in or not by using the option `allowN
   };
 ```
 
+### Anonymous User upgrade Settings
+When an anonymous user signs in or signs up with a permanent account, the `allowAutoUpgradeAnonymousUsers` option allows you to link the existing account (anonymous) with permanent account. This way the user can continue with what they were doing before signing up. By default this option is disabled.
+```javascript
+  const config = {
+    ...
+    allowAutoUpgradeAnonymousUsers: true,
```

Comment:
@mjsirisha88, for the config parameters, I have been using the same name of the iOS properties. Would it make sense if we rename it simply to `autoUpgradeAnonymousUsers` to follow that personal convention?